lib/commands/serve.js

```diff
@@ -41,8 +41,10 @@ module.exports = Command.extend({
     return this._checkExpressPort(commandOptions)
       .then(this._autoFindLiveReloadPort.bind(this))
       .then(function(commandOptions) {
+        var config = this.project.config(commandOptions.environment);
+
         commandOptions = assign({}, commandOptions, {
-          baseURL: this.project.config(commandOptions.environment).baseURL || '/'
+          baseURL: config.rootURL || config.baseURL || '/'
```

Comment:
adjusted in 8d2d839, though I'm not quite convinced if that really has any substantial benefit